spec/descriptions/info.md

```diff
@@ -1,197 +1,38 @@
-
-The data field is optional and can contain arbitrary key-value pairs. The behavior of supplying duplicate keys is unspecified.
-
-The error field is optional and can be set to true to indicate an erroneous span.
-
-The endpoint also accepts a batch of spans, which then need to be given as an array:
-
-```json
-[
-  {
-    // span as above
-  },
-  {
-    // span as above
-  }
-]
-```
-
-For traces received via the Trace SDK Web Service the same rules regarding [Conversion and Service/Endpoint Naming](https://www.ibm.com/docs/en/obi/current?topic=references-java-trace-sdk#conversion-and-naming) are applied as for the Java Trace SDK. In particular these key-value pairs in data are used for naming: service, endpoint and call.name.
-
-Note: The optional [Instana Agent Service](https://www.ibm.com/docs/en/obi/current?topic=requirements-installing-host-agent-kubernetes#instana-agent-service) provided on Kubernetes via the Instana Agent Helm Chart is very useful in combination with the Trace Web SDK support, as it ensures that the data is pushed to the Instana Agent running on the same Kubernetes node, ensuring the Instana Agent can correctly fill in the infrastructure correlation data.
-
-#### Curl Example
-
-The following example shows how to send to the host agent data about a matching ENTRY and EXIT call, which simulates a process that receives an HTTP GET request targeted to the https://orders.happyshop.com/my/service/asdasd URL and routes it to an upstream service at the https://crm.internal/orders/asdasd URL.
-
-```bash
-#!/bin/bash
-
-curl -0 -v -X POST 'http://localhost:42699/com.instana.plugin.generic.trace' -H 'Content-Type: application/json; charset=utf-8' -d @- <<EOF
-[
-  {
-    "spanId": "8165b19a37094800",
-    "traceId": "1368e0592a91fe00",
-    "timestamp": 1591346182000,
-    "duration": 134,
-    "name": "GET /my/service/asdasd",
-    "type": "ENTRY",
-    "error": false,
-    "data": {
-      "http.url": "https://orders.happyshop.com/my/service/asdasd",
-      "http.method": "GET",
-      "http.status_code": 200,
-      "http.path": "/my/service/asdasd",
-      "http.host": "orders.happyshop.com"
-    }
-  },
-  {
-    "spanId": "7ddf6b31b320cc00",
-    "parentId": "8165b19a37094800",
-    "traceId": "1368e0592a91fe00",
-    "timestamp": 1591346182010,
-    "duration": 97,
-    "name": "GET /orders/asdasd",
-    "type": "EXIT",
-    "error": false,
-    "data": {
-      "http.url": "https://crm.internal/orders/asdasd",
-      "http.method": "GET",
-      "http.status_code": 200,
-      "http.path": "/orders/asdasd",
-      "http.host": "crm.internal"
-    }
-  }
-]
-EOF
-```
-
-#### Limitations
-
-Adhere to the following rate limits for the trace web service:
+### Rate Limiting
+A rate limit is applied to API usage. Up to 5,000 calls per hour can be made. How many remaining calls can be made and when this call limit resets, can inspected via three headers that are part of the responses of the API server.
```

Comment:
can **be** inspected

Reply:
@FarhadInstana - I'm going to add that into my next set of revisions, thank you!